docker/docs/startup.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# Clone Chain Core
+git clone https://github.com/chain/chain.git $CHAIN
```

Comment:
That would sure be faster. I wanted to avoid requiring a local checkout, but that's silly once you're running a script from the repo.